### ISSUE TITLE
Disable bold highlight

### ISSUE THREAD

**Issue by uloco** [jesseduffield/lazygit#2304]
Is there a way to disable the bold highlighting of selected lines?
I just want it to reverse and not bold the text.
I tried doing so with the config but whatever color/modifier I use the text is is always bold. (and uses the bold color)

**Comment by svanharmelen** [comment]
Did you ever find a solution @uloco? Would like to "fix" that as well :)

**Comment by uloco** [comment]
Idon't remember but I think I worked around it.

See my dotfiles for config

**Comment by svanharmelen** [comment]
Will have a look, thanks...

**Comment by niksingh710** [comment]
@svanharmelen got solution?

**Comment by stefanhaller** [comment]
It's [hard-coded in gocui](https://github.com/jesseduffield/gocui/blob/86e0d5a312de1ba0b89435584fc990beb53448b2/view.go#L487C28-L487C32) right now, there's no way to change it through configuration. That would be a feature request, I guess.

**Comment by svanharmelen** [comment]
@niksingh710 not really, but I added this to my `lazygit.yml` which made the "problem" much less visible (selected lines looks much more natural now). So for me this is not really an issue anymore...

```
gui:
  theme:
    selectedLineBgColor:
      - '#444444'
```

**Comment by tasuki** [comment]
Ah I'd also love to be able to make the highlight non-bold!
